.github/workflows/publish_jobs.yml

```diff
@@ -0,0 +1,34 @@
+# This workflow publishs all Jenkins jobs when a PR is merged.
+#
+---
+on:
+  push:
+    branches:
+      - main
+    paths:
+      # Run only when the definitions of jobs changed.
+      - 'jobs-builder/jobs/**'
+name: Publish Jenkins jobs
+jobs:
+  publish:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Install the Jenkins Job Builder
+        run: |
+          pip install --user jenkins-job-builder
+      - name: Run the publish script
+        env:
+          JENKINS_USER: ${{ secrets.JENKINS_USER }}
+          JENKINS_PASSWORD: ${{ secrets.JENKINS_PASSWORD }}
+        working-directory: jobs-builder
+        run: |
+          cp jjb.conf.template jjb.conf
+          if [[ -z "$JENKINS_USER" || -z "$JENKINS_PASSWORD" ]]; then
+              echo "Missing the secrets JENKINS_USER and/or JENKINS_PASSWORD"
```

Comment:
Nit: I'd consider an `ERROR:` prefix and writing the error to `stderr`.

Reply:
Done. I was curious if sending the output to `stderr` would make any difference (i.e. github would send the message to a seperate log file) but it seems not.